customerio/api.py

```diff
@@ -0,0 +1,15 @@
+"""
+Implements the client that interacts with Customer.io's App API using app keys.
+"""
+from .client_base import ClientBase
+
+class APIClient(ClientBase):
+    def __init__(self, key, url=None, retries=3, timeout=10, backoff_factor=0.02):
+        self.url = url or 'https://api.customer.io'
+        ClientBase.__init__(self, retries=retries,
+                            timeout=timeout, backoff_factor=backoff_factor)
+        self.http.headers = {
+            "Authorization": "Bearer {key}".format(key=key)}
+
+    def send_email(self, **kwargs):
```

Comment:
Hmm, using this signature it'd error out (at least in Python 2.x) because the following contains a Python keyword `from`:
`send_email(to="email@org.com", from="some@from.com", transactional_message_id=23, customer_id=23)`

Or we can do this:

```def send_email(self, data)```
and then call it as 

data = {
    "to": "waqas7@customer.io",
    "from": "something@customer.io",
    "customer_id": "20201126030726027693",
    "transactional_message_id": 70}

api.send_email(data)
```

Reply:
good catch with the `from` reserved keyword, doesn't work in python 3 either (tried with 3.6.9)

alternatively if the message is defined in a dict can use `**` to expand it so the usage would be `api.send_email(**data)`

Reply:
👍 Either works—message being defined before invocation is also ideal for the way our code snippets work in the UI.